modules/adding-node-iso-flags.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// *nodes/nodes/nodes-nodes-adding-node-iso.adoc
+
+[id="adding-node-iso-flags{context}"]
+= Adding a node with command flags
+
+You can add a single node to your cluster by using command flags to specify configurations for the new node.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`)
+* You have an active connection to your target cluster
+* You have a kubeconfig file available
+
+.Procedure
+
+. Generate the ISO image by running the following command. The MAC address must be specified using a command flag.
++
+[source,terminal]
+----
+$ oc adm node-image create --kubeconfig='<path_to_kube_config>' --mac-address=<mac_address>
+----
++
+--
+where:
+
+`<path_to_kube_config>`:: Specifies the path to the kubeconfig file.
+`<mac_address>`:: Specifies the MAC address of the node that is being added.
+--
++
+[TIP]
+====
+To see additional flags that can be used to configure your node, run the following command:
+[source,terminal]
+----
+$ oc adm node-image create --help
```

Comment:
I think it might be helpful to still urge users run the help flag so they can see the flags within the OC command, but I'm also willing to try adding a reference of the configuration flags available to users somewhere in this doc, perhaps below this specific procedure